spec/fixtures/sign_in/oauth.key

```diff
@@ -1,27 +1,28 @@
------BEGIN RSA PRIVATE KEY-----
-MIIEowIBAAKCAQEA10sRGVeQDgXCPJhQIUvUbKrRLdf5JqEFkscniTW2pNd+o6nD
-VNtqM09TsVgXy59UIvPh/wzl4KcJQOLnLPZnJt9pkDnw3mBvJy+GXPAIhtUYaeTE
-HhEcWlpW1qm58euYSD4QoTRXrQoXeumPLNUfQ7sWXmoMBTwkXMJNynpl0drRHfFu
-BpHEAyZHpF45tIH7qKELCWt17tx7YokCmbIqDotY9rbABxOwpX5xiGyZNuqFOiSu
-K8qcXG1nuS+GwBGHg9Pp8u549Jpn41kCGdD7bYwO7+zVwPaK/7PFz8CmxbObFTkZ
-n/yhFYSel3RHuuRH/RLN0xLhvMbGxKfLCulYmQIDAQABAoIBAFb6h9zyKEvR/cgX
-TQA2r4p7dGkOASWfFi5XcVyYq8JrIUmjszucLYqvOMPEMFDefNJmL16oalLZiP6C
-7RyYK9JjnjVbgzmEQkPVinYPr5Ixk1xsxOLUzNyf9Mdi6LGZGfvNv1AxYl6RXi0V
-IPfe3nOxqPbZ5yNSSeqDwYPU8T/XZsFrd/cIM2z8tAL+pW/YTQ6HbPyuLZLivtRq
-bJInB+uCXSZB1qKjnOIY/nNNFe2nypoJs011yMvcRW3B+VRdjiSNkqLXN7zwBy/U
-CafQ337TjRKO1gM3RA7dOuIhKOYd/cZu1/ch54jCR88Cdj8VKKmfHR+HkRCWxOhj
-38e0Z1UCgYEA5/Ff5NxuQ43dj2dXe+HnLpee09ZAMUFnwsZP71Be6ZwAIhS6OfhB
-GCLYNlzQwkgjXtx6j7watzC4QqKgwhcr3uktlWxMr6Lj4Mnn8rczaX6NNwoh1ptv
-PI5g5xNt9/mbenYg6O+ra0AbCZMrtQFLNsGa8GRFwQveyofpSmMwgusCgYEA7Z+a
-+AMfo7fkFk8K5Rer1p7ujdS+Ajuwq5W5G/uVHV0p1XbCiAuIaXIRjxA7NSrEA8NU
-Jov8ms/JNGwpPFExXDNQC1ROtO0fEy6Q07YDYRIN0j2HO3+hDTw4VOxWcZTi+q/Q
-w108GM+uPoRh5Kci5eieY0fyDAcCGIlGSgLoCYsCgYEArmhSM0WNpD6I28xZfE3i
-CKb6R1BT9LW50DxO3yUQOZmUgAwV/IEwcXXCHx6r22tjLXGct3g6noWECJdBSrEh
-y/yKylElWOyjbeRlBZ28ixceGMUFrNo0StTFCP0TM+P9hZhOJtoRjBQoQIrp0rGm
-XQY3+4tq6D6EyMZqJgVv9QECgYByFtGnBojoaGBgva6zc/wTXdeXzW81T0dx+r8X
-ymE76gEt9olMHI1wY04+7EIhMtlOHTiIEWpFQkKOoufhh/I1HIXwCpiHkdn7sb3j
-iF5y+uIb8WK+lHbVDfbV/2s5uYyJZGTNiDRUjVJRvenqiLzeuiNh86cpUywUb8SJ
-wH98JQKBgAe7QOLbSWeSoFNHFqqFoU4ex+ag7KqP3uNdpfwwP78kqVzu6X1FpUTD
-pntrAmN+8UNHLadDiPMImka73wID0GIrpE/0WIJATX3SolXcQWU2nFfOK9MS8tej
-yFE2MgtwvnFxCM44DAgyjmEYZKYHchRNil2sAnznve9ud2y1zZdv
------END RSA PRIVATE KEY-----
+-----BEGIN PRIVATE KEY-----

```

Comment:
🗣️ Does this change matter? Asking because I honestly don't know the difference between a private key vs an RSA private key in this context.

Reply:
yea it doesnt matter, just make it whatever the raw was that you received. Ive seen both come from venafi for no rhyme or reason